README.md

```diff
@@ -12,6 +12,7 @@ This bot can be run with Docker or simply `npm start`. In both cases `.env` need
 ```
 NODE_ENV=production
 BOT_TOKEN=<TELEGRAMBOTFATHERTOKEN>
+BOT_USERNAME=<TELEGRAMBOTUSERNAME>
```

Comment:
So why is this needed now? It could have previously been read from `bot.options.username = botInfo.username`.

Reply:
It is required because the information is required for creating the bot instance before you can even access `botInfo.username`

Reply:
Alright, I guess that's what's meant here in the documentation: http://telegraf.js.org/#/?id=command-handling-in-group

Reply:
correct